swatch-core/schemas/utilization_summary.yaml

```diff
@@ -0,0 +1,17 @@
+$schema: http://json-schema.org/draft-07/schema#
+title: UtilizationSummary
+description: Contains tally summary data enriched with subscription capacity information
+properties:
+  org_id:
+    description: The identifier for the relevant organization.
+    type: string
+  product_id:
+    type: string
```

Comment:
I'm ok to use this utilization_summary object for now, since the good one will be merged from https://github.com/RedHatInsights/rhsm-subscriptions/pull/5114. 
But will put on hold label and a comment that this pull request depends on https://github.com/RedHatInsights/rhsm-subscriptions/pull/5114.
Note that the implementation will need to change accordingly.